src/synthesizer/sed.py

```diff
@@ -439,6 +435,33 @@ def shape(self):
         """
         return self.lnu.shape
 
+    @property
+    def bolometric_luminosity(self):
+        # Calculate the bolometric luminosity using the trapezium rule.
+        # NOTE: to use an alternative integration method
+        # measure_bolometric_luminosity method can be used instead.
+        # NOTE: the integration is done "backwards" when integrating over
+        # frequency. It's faster to just multiply by -1 than to reverse the
+        # array.
+        integral = -integrate_last_axis(
+            self._nu,
+            self._lnu,
+            nthreads=1,
+            method="trapz",
+        )
+
+        # return the bolometric luminosity with units
+        return integral * self.lnu.units * self.nu.units
```

Comment:
Here is where you could instantiate a `Units` object and extract the bolometric luminosity units for conversion. However, it would be weird for the user to set a system where the combination of `lnu` and `nu` units didn't give the bolometric luminosity 😅 I'm tempted to say we leave it like this?

Reply:
Do you mean e.g.:

```
from synthesizer.units import Units
units = Units()
units.luminosity
```
or is there a faster way of extracting the units?

Incidentally, in an ideal world I think we would have just defined the units of `luminosity` and then aliased `bolometric_luminosity` to it.

Reply:
That would be the way to do it. However, I've wanting for a while now to categorise the unit system so that the user defines `spatial`, `luminosity`, `wavelength` etc. and then those get assigned to the specific attribute within `Units` (or aliased as you say).

If I did that then the code you have here is already sufficient.